gateway/1.4/modules/ROOT/pages/flex-gateway-k8-external-dns.adoc

```diff
@@ -0,0 +1,119 @@
+= Provide a Domain Name for Your Kubernetes Service
+
+Use ExternalDNS to expose your Kubernetes service through a domain name instead of a difficult-to-understand external IP or random host name. 
```

Comment:
@ahejeij  Is "Kubernetes service" ok? What do we mean exactly?

Reply:
It refers to your kubernetes application let's say. You have an app (or service) that you want to expose through flex gateway